### DISCUSSION TITLE
v7: Running `npm install` updates package.json timestamp

### DISCUSSION THREAD

**Discussion by coreyfarrell** [npm/feedback#226] [CLI] [answered]
When running `npm install` with v7 the timestamp of `package.json` is updated, even when running install with no additional arguments.  So far I have not seen this alter the contents of the file.  This can be problematic if other programs are monitoring that file for changes, for example my text editor prompts if I want to reload the file from disk.

**Comment by MylesBorins** [comment, marked as answer]
@coreyfarrell could you open this as an issue on the CLI repo? This seems like something the @npm/cli team should triage over there

**Reply by coreyfarrell** [reply]
thanks @MylesBorins, reposted to https://github.com/npm/cli/issues/2714.
